Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -1680,7 +1680,7 @@ int CLuaElementDefs::DestroyElement(lua_State* luaVM)
     if (!argStream.HasErrors())
     {
         // Destroy it
-        if (CStaticFunctionDefinitions::DestroyElement(*pEntity))
+        if (pEntity->CanBeDestroyedByScript() && CStaticFunctionDefinitions::DestroyElement(*pEntity))
```

Comment:
Wouldn't it make more sense to add this to `CStaticFunctionDefinitions::DestroyElement` instead? Seems a bit odd to me to have it as part of the Lua interface

Reply:
done